connectors/sources/tests/test_mysql.py

```diff
@@ -188,6 +246,7 @@ async def test_ping_negative(patch_logger):
             await source.ping()
 
 
+@pytest.mark.fail_slow(10)
```

Comment:
This is intentional and will be fixed with a separate PR. Test was always this slow, but shouldn't be the focus in this PR.